test/test.ts

```diff
@@ -107,13 +107,22 @@ describe('cloud functions api', () => {
 });
 
 describe('end to end', () => {
-  it('should work together end to end', async () => {
+  it('should deploy end to end', async () => {
     const scopes = [mockUploadUrl(), mockUpload(), mockDeploy(), mockPoll()];
     const projectId = 'el-gato';
     const deployer = new gcxp.Deployer({name, targetDir, projectId});
     await deployer.deploy();
     scopes.forEach(s => s.done());
   });
+
+  it('should call end to end', async () => {
+    const scopes =
+        [mockUploadUrl(), mockUpload(), mockDeploy(), mockPoll(), mockCall()];
```

Comment:
You don't need all these mocks for call :)  Should just be `const scopes = [mockCall()];`

Reply:
In fact, the other mocks seem to break the test.